src/settingsdialog.ui

```diff
@@ -23,8 +23,8 @@
       <attribute name="title">
        <string>General</string>
       </attribute>
-      <layout class="QGridLayout" name="gridLayout" rowstretch="0">
-       <item row="0" column="0">
```

Comment:
Do you know why there is a row change here?

Reply:
No idea, perhaps I clicked on an object and Qt Creator changed it for some reason.